flowetl/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+black = "==19.3b0"
+apache-airflow = "==1.10.3"
+pylint = "*"
```

Comment:
We don't use pylint anywhere else, can we get the reason we're using it here on the record?

Reply:
Because I do use it...

Reply:
It's a dev dep I use it during dev - it doesn't hurt anyone by it being there. If it's not it makes it a bit more painful for me...